worker/worker_test.go

```diff
@@ -96,6 +97,20 @@ func (w *testWorker) blockUploads() func() {
 	return func() { close(blockChan) }
 }
 
+func (w *testWorker) blockAsyncPackedSlabUploads(up uploadParameters) {
```

Comment:
I commented on this in the other PR already but I'd like to expose methods like this to differentiate between internal methods of the test framework and the ones to use in tests.